### ISSUE TITLE
[v2, Win10x64] Resizable window has no resize controls at window frame border

### ISSUE THREAD

**Issue by erandelax** [wailsapp/wails#1062]
**Description**
Despite window being marked as `options.App{DisableResize:false,Frameless:false}` there are no size controls at the window frame borders (cursor stays default on hover). Resizing via maximize button at app title bar works fine, but it is impossible to adjust window size by manually dragging its corner.

Default window dropdown menu resize action (that appears when you RMB on app icon at app title bar) does not seem to work either.

UPD. Looked into e9aba4795f6b2341f8242dab36cd631a36b79c7e, seems like `window.wails.flags.enableResize == false`  on app start. So far partially resolved the issue setting it to true manually (seems like it mimicks default resize behavior through `body` events at frontend side, not exactly what needed but... it works).

**To Reproduce**
Appears at any app served right after `wails init`.

**Expected behaviour**
Window size to be adjustable by dragging in and out window frame.

**System Details**
```
System
------
OS:             Windows 10 Pro
Version:        2009 (Build: 19043)
ID:             21H1
Go Version:     go1.17.5
Platform:       windows
Architecture:   amd64

Dependency      Package Name    Status          Version
----------      ------------    ------          -------
WebView2        N/A             Installed       96.0.1054.62
npm             N/A             Installed       8.3.0
*upx            N/A             Available
```

- [X] This issue is for Windows and I have read the [Windows Developer Guide](https://wails.app/guides/windows/)

**Comment by leaanthony** [comment]
There's a fix on `master` and already a ticket for this. Please test.

**Comment by leaanthony** [comment]
Actually, the fix isn't complete. I've tried finding the commit that broke this and am getting really weird local behaviour: if I use the published beta.21 version, generate a project, it works. If I do the same with beta.22, it doesn't. If I go anywhere between and use replace, it doesn't work - not even for the commit that is beta.21.... If you can find the commit that broke it, we can fix it.

**Comment by leaanthony** [comment]
This is fixed in `v2.0.0-beta.28`
